### ISSUE TITLE
Some manga cannot added to "device list"

### ISSUE THREAD

**Issue by AdityaMH** [KotatsuApp/Kotatsu#598]
### Brief summary

Some manga can't added to "device list" even from downloaded or importing.

### Steps to reproduce

1. Download all chapter [this](https://comick.app/comic/amagami-sincerely-yours)
2. Downloaded, added to the download list
3. Go to "on device"
4. Manga unavailable

### Kotatsu version

6.5 (but i'm download this manga at version 597)

### Android version

10

### Device

Android

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] If this is an issue with a parser, I should be opening an issue in the [parsers repository](https://github.com/KotatsuApp/kotatsu-parsers/issues/new/choose).

**Comment by AdityaMH** [comment]
download are gone now. i can't read comic

**Comment by Koitharu** [comment]
Is it still happens on v6.5.4?

**Comment by AdityaMH** [comment]
yes, because manga was deleted or cannot accessed from website source. (something like blocked from server)

**Comment by AdityaMH** [comment]
only one solution is only redownloading from other source.
